### DISCUSSION TITLE
How to reload system hostname?

### DISCUSSION THREAD

**Discussion by troglobit** [lldpd/lldpd#588] [Q&A]
Hi, this is related to #549.  I have a system based on NETCONF/sysrepo that can update the system hostname (`sethostname()`, `/etc/hostname` and `/etc/hosts`) from under a running lldpd.

Is there any way to tell lldpd to reload (`SIGHUP`) common information like this from the system without stop/starting lldpd?  And yes, I know of `lldpcli configure system hostname foobar`, just curious if there is any other mechanism I have not yet discovered.

**Comment by vincentbernat** [comment]
I think `lldpcli update` should do that too.

**Reply by troglobit** [reply]
Aha, missed that command. Unfortunately it doesn't update the hostname (only sends out new frames).
